examples/custom-usage-counter/custom_usage_counter.py

```diff
@@ -57,6 +57,9 @@ async def shutdown(self):
             yield s
         print(f"service {self.id} stopped on '{self.provider_name}'")
 
+    async def reset(self):
+        """Do nothing when the service is restarted"""
```

Comment:
I'd add this as a _comment_ not as a docstring ... but maybe that's just me... 

anyway, I'd rather accent the point that we don't _need_ to do anything on restart